src/templates/homepage/HeroSection/HeroSideLayout.tsx

```diff
@@ -0,0 +1,180 @@
+import editorStyles from "styles/isomer-cms/pages/Editor.module.scss"
+
+import { getClassNames } from "templates/utils/stylingUtils"
+
+import { SectionBackgroundColor, SectionSize } from "types/hero"
+
+const TRANSLUCENT_GRAY = "#00000080"
+
+interface HeroInfoboxProps {
+  title: string
+  subtitle?: string
+  url?: string
+  button?: string
+}
+const HeroInfobox = ({ title, subtitle, url, button }: HeroInfoboxProps) => {
+  return (
+    <div className={getClassNames(editorStyles, ["py16"])}>
+      <div className={getClassNames(editorStyles, ["mb8"])}>
+        <h1 className={getClassNames(editorStyles, ["hero-title", "mb4"])}>
+          {title && (
+            <>
+              <b className={getClassNames(editorStyles, ["is-hidden-touch"])}>
+                {title}
+              </b>
+              <b className={getClassNames(editorStyles, ["is-hidden-desktop"])}>
+                {title}
+              </b>
+            </>
+          )}
+        </h1>
+
+        {subtitle && (
+          <p
+            className={getClassNames(editorStyles, [
+              "is-hidden-touch",
+              "hero-subtitle",
+            ])}
+          >
+            {subtitle}
+          </p>
+        )}
+
+        {url && button && (
+          <a
+            href="/"
```

Comment:
Might be a good idea to remove so users don't accidentally click the links?

Reply:
oh dead i realised we silenced the error on `jsx-ally` in our legacy code. i'll change ._.